client/src/App.css

```diff
@@ -1,38 +1,273 @@
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+body {
+  font-family: "Raleway", sans-serif;
+  font-size: 16px;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+/*Primary Colors */
+
+.primary-pink {
+  background-color: #fae5dc;
+}
+
+.secondary-pink {
+  background-color: #f9dacd;
+  opacity: 0.4;
+}
+
+/* black for heading or background */
+.black {
+  background-color: #000000;
+}
+
+/*Other Colors */
+.page-background-grey {
+  background-color: #e5e5e5;
+}
+
+.brown-background {
+  background-color: #441f0e;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+/* Desktop Appointment Flow */
+
+.desktop-appointment-beige {
+  background-color: #441f0e;
+  opacity: 0.2;
+}
+
+.desktop-appointment-grey {
+  background-color: #e2dfdf;
+}
+
+/* Mobile Appointment Flow */
+
+.mobile-appointment-grey {
+  background-color: #441f0e;
+  opacity: 0.1;
+}
+
+/* Typography */
+
+h1 {
+  font-family: "Playfair Display", sans-serif;
+  font-style: italic;
+  font-weight: bold;
+  font-size: 45px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
+}
+
+h2 {
+  font-family: "Playfair Display", sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 40px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
 }
 
-.App-link {
-  color: #61dafb;
+h3 {
+  font-family: "Raleway", sans-serif;
+  font-weight: medium;
+  font-style: normal;
+  font-size: 24px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+h4 {
+  font-family: "Raleway", sans-serif;
+  font-weight: bolder;
+  font-style: normal;
+  font-size: 30px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
+}
+
+h5 {
+  font-family: "Raleway", sans-serif;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 24px;
+  line-height: normal;
+  letter-spacing: 0;
+  color: #000000;
+}
+
+.body1 {
```

Comment:
Can you come up with more semantic names? We typically never want to put numbers in class names